### DISCUSSION TITLE
如何爬取分页页面或者控制页面

### DISCUSSION THREAD

**Discussion by binarytree** [coder-hxl/x-crawl#114] [Q&A]
控制页面
爬取动态页面支持自动化操作、键盘输入、事件操作等。

可以使用上面的机制么，具体如何使用

**Comment by coder-hxl** [comment]
https://coder-hxl.github.io/x-crawl/cn/guide/crawler-style#example
看传统爬虫那部分

**Comment by coder-hxl** [comment]
https://coder-hxl.github.io/x-crawl/cn/guide/crawl-page
crawlPage API 跟 puppeteer 的关系：https://coder-hxl.github.io/x-crawl/cn/about/faq#the-relationship-between-crawlpage-api-and-puppeteer
